docs/ref/contrib/auth.txt

```diff
@@ -268,6 +268,11 @@ Manager methods
         :class:`~django.contrib.auth.models.User` object will have
         :attr:`~django.contrib.auth.models.User.is_active` set to ``True``.
 
+        Note that the default implementation of `create_user` requires
```

Comment:
This should say something like... 

`email` is defined as an optional keyword argument but, unless overridden, is required by `AbstractUser.REQUIRED_FIELDS`. (Link to :attr:`~django.contrib.auth.models.CustomUser.REQUIRED_FIELDS`  in `customizing.txt`)

Drop the Note that, and put it in an `.. admonition:`

Reply:
Changed.

The "coding" in documentation is often as challenging as the code itself. But I love how it takes care of formatting itself. Just have to make sure to pay attention to the double colons and double backticks.